package.json

```diff
@@ -34,7 +34,6 @@
   "dependencies": {
     "@google-cloud/projectify": "^0.3.3",
     "@google-cloud/promisify": "^0.4.0",
-    "@types/duplexify": "^3.6.0",
     "@types/request": "^2.48.1",
```

Comment:
Can this be removed as well (okay if in a different PR)?

Reply:
That's my plan.  It is more complex, so I am doing it in a follow up PR.